### ISSUE TITLE
Alexa App delayed or no respond

### ISSUE THREAD

**Issue by thamaddin** [alexa/alexa-avs-sample-app#889]
I have troubles running the alexa-avs-sample-app on a Raspberry Pi 3.

In general it seems everything is working. Node server is listening on port 3000. Java client is working.
The WakeWord Enginge is working fine. After I asked a question the audio recording is not stopping or with a delay of a few seconds. Sometimes I get an answer but more often not.  

I'm using a Logitech USB Webcam as microphone and the 3,5mm jack for audio out

Do you have any suggestion what I can test or do?

**Comment by AlphaDrakonix999** [comment]
Hello Thamaddin,

Thank you for reaching out to us with your inquiry.  If the audio recording is not stopping then it could be that there is some background noise that keeps the microphone active.  We would suggest to:

+ Try using a noise-cancelling microphone headset to test your setup.
+ Try adjusting the microphone sensitivity so that background noise won't keep it active.
+ Try disabling any microphone-sensitivity auto-adjustment feature.

Please let us know if this information was helpful to solve your issue.

Best,

**Comment by daverathbone** [comment]
**_No this is not going to fix it!_**

The wake system is over powered by using Java on the Pi.
I am working on a complete re-write that just uses Node.js. 
Node is none blocking and can handle all the different audio channels and web services.
The good thing is you can use Bing (_err_ "Amazon") to get smart stuff an convert keywords to IOT local devices. Don't bother with add on's to this code , if so your better buying a Dot!
Your welcome to contact me for further detail.

What does work Amazon AWS with Lambda for remote access to your Pi dot clone (with Node.js)
or Echo dot with your own skill's. But this code is way to buggy to use in any real application.

**Comment by thamaddin** [comment]
Thanks for your answer.

I investigated that there is a problem with my local broadband connection. When I use my mobile phone as hotspot for the Pi, the avs sample works fine and is fun to use. 
I connected my Pi directly to my local broadband connection (no other components) and even this was not working properly (very laggy or no respond)

I compared the ping times between my broadband connection and mobile internet to the avs server. As logically expected the times for broadband connection were smaller. I don't know why this connection works so bad with the avs sample.
